pkgs/misc/drivers/epson-escpr/default.nix

```diff
@@ -29,7 +30,14 @@ in
 	    enable = true;
 	    drivers = [ pkgs.epson-escpr ];
 	  };
-      '';
+
```

Comment:
This snippet is in this file for ages.

I agree that this ought to be moved to the manual which currently totally lacks any information about printing/CUPS/whatever. And I am not sure I am a right person to do that because I use NixOS for like 5 days :)